applications/logan_progress_report/logan_progress_report.org

```diff
@@ -0,0 +1,275 @@
+#+LATEX_HEADER: \input{styles.tex}
+#+TITLE: hackEDU Progress Report
+* Accomplishments to Date
+** Hack Camp
+We took this summer to run Hack Camp, a two-week summer program that brought
+local high school students from little or no coding experience to publishing
+their first websites and games. In running Hack Camp we had two goals:
```

Comment:
`In running Hack Camp we had two goals` => `We had two goals in running Hack Camp`